### PR TITLE
Srinter 749

### PR DESCRIPTION
**Pull request by soda736**
Cancelled my last pull request as I made another commit which didn't show up in the original pull request. New pull request has all 3 commits.

### CODE REVIEW

File: config/sync/block.block.mainnavigation_2.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ dependencies:
 id: mainnavigation_2
```
Comment (omahm) [inline config/sync/block.block.mainnavigation_2.yml:13]: Can you raise a ticket to see about renaming this to something like sidebar_mainnavigation, at present it's a little ambiguous. The machine name should be generated from the block description.

**Review by omahm — COMMENTED** [review]
Looks good to me.

File: config/sync/block.block.mainnavigation_2.yml (hunk 1 above)
Comment (ooneill547) [reply config/sync/block.block.mainnavigation_2.yml:13]: That will be my fault, I'll see about updating the machine names

Comment (omahm) [reply config/sync/block.block.mainnavigation_2.yml:13]: Not a fault, just something you pick up along the journey.

**Pull request merged**